src/main/java/org/folio/service/transactions/batch/BatchPaymentCreditService.java

```diff
@@ -73,6 +73,8 @@ private void updateEncumbranceToCancelTransaction(Transaction transaction, Trans
     // NOTE: we can't unrelease the encumbrance automatically because the payment/credit does not say if the encumbrance
     // was released automatically or not (as opposed to a pending payment).
     double expended = encumbrance.getEncumbrance().getAmountExpended();
+    double credited = encumbrance.getEncumbrance().getAmountCredited();
+    expended = subtractMoney(expended, credited, currency);
```

Comment:
same here, try to keep expended and credited separate

Reply:
THanks, fixed